style: Update interface of badges in repository component

rkpasia opened the pull request:
<!--
  Bonjour!

  We can't express how grateful we are that you're working on making GitPoint
  better! We're thrilled to take a look at the changes you've made and merge
  them in as soon as possible. Please fill out this template to make the
  reviewal process as quick and smooth as possible. In addition, please make
  sure you remember to add yourself to the contributors list with the following
  command:

  $ yarn contributors:add

  Make sure the title of your PR follows our commit style guidelines (see other
  open PR's for reference if you're confused):

  https://github.com/angular/angular.js/blob/master/DEVELOPERS.md#-git-commit-guidelines

  Thanks again for your hard work!
-->

| Question         | Response    |
| ---------------- | ----------- |
| Version?         | v1.4.1      |
| Devices tested?  | iPhone 6 (Emulator) |
| Bug fix?         | no      |
| New feature?     | no      |
| Includes tests?  | no      |
| All Tests pass?  | yes      |
| Related ticket?  |         |

---

## Screenshots

<!--
  Replace the images in the table below with screenshots of your changes before
  and after. If this is not applicable (i.e. absolutely NO visual changes), feel
  free to delete this section.
-->

| Before   | After    |
| -------- | -------- |
|![before](https://user-images.githubusercontent.com/5786033/35542960-e7b1f0b0-0562-11e8-97df-8acf71aaefc3.png)|![after](https://user-images.githubusercontent.com/5786033/35542997-1842a54e-0563-11e8-88cd-eea15b3d9197.png)|

## Description
Hello! In this short commit I changed the styles associated with the badges in the repository component. I wasn't happy with the previous interface, so I tried to play with it and the results can be seen in the screenshots :)

I haven't created an issue or discussed with anyone else about this before doing it. So if it isn't accepted I understand, it just seemed a good way to start contributing to the project :)


<!-- DO NOT MODIFY BELOW THIS LINE -->
<!-- ----------------------------- -->
<!-- GITPOINT_PR -->

machour commented on src/components/repository-profile.component.js:
```diff
@@ -98,18 +98,18 @@ const styles = StyleSheet.create({
-    marginLeft: 27,
-    marginRight: 27,
-    borderWidth: 0.5,
+  badge_view: {
```
badgeView would be me consistent with our code style

machour commented on src/components/repository-profile.component.js:
```diff
@@ -98,18 +98,18 @@ const styles = StyleSheet.create({
+  },
+  badge: {
+    color: colors.white,
+    fontWeight: 'bold',
```
Use ...fonts.fontPrimaryBold instead

machour commented:
Hey, this is a fantastic way to start contributing, what a nice UI update :tada: :tada:

Left you some minor changes requests to stick to our coding habits. 

Could you also add yourself as a contributor using `yarn contributor:add` ? 

Thank you again for your PR!

housseindjirdeh commented:
This is so great @rkpasia 😍 . I knew those tags could use a little bit of love, but couldn't place my finger on what exactly it needed to look a little better.

Aside from the tiny things @machour, this looks great. Also, please don't forget to add yourself as a contributor with `yarn contributors:add`!!

rkpasia commented:
@machour thanks for the feedback! 😄 In the last commit I've updated the code for consistency.

@housseindjirdeh Thanks for the great words 😄I've added myself to the contributors list! Forgot about it ahah

I'll reach out in gitter about the possibility to further get involved in the project 😄
